MODULE_EVALUATION_TEMPLATE.MD

```diff
@@ -21,7 +21,6 @@ When performing a technical evaluation of a module, create a copy of this docume
 * [ ] Module build MUST produce a valid module descriptor
 * [ ] Module descriptor MUST include interface requirements for all consumed APIs
 * [ ] Third party dependencies use an Apache 2.0 compatible license
-* [ ] In order to ensure reproducible builds, snapshot versions of build-time dependencies should not be referenced.
```

Comment:
I suggested in TC just now that we leave the criteria there (so that we are not ignoring a criterion), but change it to mirror what we say about justifying when we disable/ignore Sonarqube issues.  So suggesting a change to:

> In order to ensure reproducible builds, snapshot versions of build-time dependencies should be reasonably justified.